### PR TITLE
Custom internal alias

### PR DESCRIPTION
**Pull request by satr**

### CODE REVIEW

File: src/components/app-overview/index.tsx [hunk 1]
```diff
@@ -28,7 +28,13 @@ export const AppOverview: FunctionComponent<{ appName: string }> = ({
     { skip: !appName, pollingInterval: 15000 }
   );
 
-  const { appAlias, environments, jobs, registration } = application ?? {};
+  const {
+    appAlias,
+    dnsAliases: dnsAliases,
```
Comment (Pespiri) [inline src/components/app-overview/index.tsx:33]:
```suggestion
    dnsAliases,
```

File: src/components/page-active-component/dns-alias.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+export const DnsAlias: FunctionComponent<DNSAliasProps> = ({ dnsAliases }) => (
+  <>
+    {dnsAliases && (
+      <div className="grid grid--gap-x-small">
+        <Typography variant="h4">
+          DNS alias{dnsAliases.length > 1 ? 'es' : ''}
+        </Typography>
+        <List>
+          {dnsAliases?.map((dnsAlias, index) => (
```
Comment (Pespiri) [inline src/components/page-active-component/dns-alias.tsx:23]: unnecessary check, object has been checked prior to usage
```suggestion
          {dnsAliases.map((dnsAlias, index) => (
```

**Review by Pespiri — CHANGES_REQUESTED** [review]

**Review by Pespiri — APPROVED** [review]

**Review by Pespiri — DISMISSED** [review]

File: src/store/radix-api.ts [hunk 3]
```diff
@@ -2239,6 +2239,16 @@ export type ApplicationAlias = {
   environmentName: string;
   url: string;
 };
+export type DnsAliasStatus = {
+  Condition?: string;
+  Message?: string;
+};
+export type DnsAlias = {
+  Status?: DnsAliasStatus;
```
Comment (Richard87) [inline src/store/radix-api.ts:2247]: API needs to be regenerated, Status must be lowercase

**Review by Richard87 — CHANGES_REQUESTED** [review]
please regenerate radix-api client

File: src/store/radix-api.ts (hunk 3 above)
Comment (satr) [reply src/store/radix-api.ts]: fixed

**Review by Richard87 — APPROVED** [review]
lgtm

**Pull request merged**